Libraries/LibC/errno_numbers.h

```diff
@@ -99,3 +99,4 @@
 #define EPROTO 71
 #define ENOTSUP 72
 #define EMAXERRNO 73
+#define EPFNOSUPPORT 74
```

Comment:
If you add a new error, it should go before `EMAXERRNO`, and be recognized by `strerror()`

Reply:
I tried doing this. While I thought it worked at first, it actually fails with undefined references when I rebuilt. Here's my ctype.h: https://gist.github.com/Lubrsi/17b4db531338f6a53692086a47cbbd59